server/index.js

```diff
@@ -1,12 +1,29 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
 
 const app = express();
 
 app.use(express.static(`${__dirname}/../client/dist`));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
+
+app.get('/info', (req, res) => {
+  const queryStr = 'SELECT * FROM homes';
+
+  db.query(queryStr, (err, result) => {
+    if (err) {
+      res.send(`ERROR, ${err}`);
+    } else {
+      res.send(result.rows);
+    }
+  });
+});
```

Comment:
Depend on the scope of your route handler, maybe separate your request handler into another file that contains all the handler ?